### PR TITLE
Add release 22.09

### PR DESCRIPTION
**Pull request by sbernauer**
Used for internal testing. Will be replaced by 22.09 in the future

### CODE REVIEW

File: releases.yaml [hunk 1]
```diff
@@ -1,5 +1,37 @@
 ---
 releases:
+  22.09-alpha1:
+    releaseDate: 2022-09-07
+    description: Second release focusing on security and OpenShift support. Will be replaced by release 22.09
```
Comment (soenkeliebau) [inline releases.yaml:5]: Maybe something like "Alpha 1 of release 22.09 - release focus is OpenShift support and security."

Comment (sbernauer) [reply releases.yaml]: Updated, please check

**Comment by sbernauer** [comment]
Actually i changed my mind and i think it's best to put out 22.09 already. Makes it easier for writing docs, testing guides, updating demos etc.

**Comment by lfrancke** [comment]
I'm fine with merging this as is. For the future do we want to have a "beta" flag or something alon gthose lines in the release.yaml and those are only shown when someone explicitly says "--beta" or "--prerelease"?z

**Review by lfrancke — APPROVED** [review]
Are these the final released versions already or will they change again?

**Comment by sbernauer** [comment]
Good idea!
These are the versions we just released and should be final. Only exception is if we find a critical bug.

**Pull request merged**